.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,47 @@
+name: "Bug report"
+description: "Report a bug in the Elastic Security documentation"
+title: "[BUG] "
+labels: "bug"
+body: 
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Use this form to alert the [@elastic/security-docs](https://github.com/orgs/elastic/teams/security-docs) team to a documentation bug.
+  - type: textarea
+    id: doc-links
+    attributes:
+      label: Documentation links
+      description: Which docs are affected?
```

Comment:
It might be good to note that we need a version specified if the reporter is sharing a bug in ESS docs. Though, idk if that ask needs to be here or further down in the form.